app/scripts/lib/strings.js

```diff
@@ -31,6 +31,22 @@ define(function (require, exports, module) {
   t('Account picture');
   t('%(permissionName)s (required)');
 
+  // Needed for Password flow update PR #3559
+  t('Enter new password');
+  t('Once you submit your new password, you will lose any Sync data that is not on one of your devices.');
+  t('Must be at least 8 characters');
+  t('Submit new password');
+  t('Click on the link we\'ve emailed you at %(email)s within the next hour to create a new password.');
```

Comment:
Good point. I believe that should be extracted as just a `'`, but we should verify.